central/telemetry/service/service_impl.go

```diff
@@ -24,9 +28,26 @@ var (
 		user.With(permissions.Modify(resources.DebugLogs)): {
```

Comment:
The `GetTelemetryConfiguration` is to be deleted, right?

Reply:
Right, as well as `ConfigureTelemetry`. This will be covered under clean-up task, I was thinking.